test/azure/generated/AzureCompositeModelClient/azureCompositeModel.ts

```diff
@@ -85,13 +85,18 @@ class AzureCompositeModel extends AzureCompositeModelContext {
    *
    * @reject {Error|ServiceError} The error object.
    */
-  listWithHttpOperationResponse(resourceGroupName: string, options?: msRest.RequestOptionsBase): Promise<Models.ListResponse> {
+  list(resourceGroupName: string): Promise<Models.ListResponse>;
+  list(resourceGroupName: string, options: msRest.RequestOptionsBase): Promise<Models.ListResponse>;
+  list(resourceGroupName: string, callback: msRest.ServiceCallback<Models.CatalogArray>): void;
```

Comment:
remaining design decision: should the ServiceCallback receive only the original body, or should it get the new flattened response value? Keeping in mind that the HttpOperationResponse is still available by defining additional parameters on the callback.

Reply:
I'll take it that you guys are basically on board with preserving the current ServiceCallback behavior? @amarzavery